crypto/src/utils.rs

```diff
@@ -0,0 +1,205 @@
+// Copyright 2018 The Exonum Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#![allow(dead_code)] // TODO Remove after complete ECR-2518 and  ECR-2519
```

Comment:
Do you plan to make functions in this module public in a separate PR?

Reply:
No I don't. The module will contain two public functions as is now.